init.lua

```diff
@@ -82,81 +96,66 @@ require('lazy').setup({
     dependencies = { 'hrsh7th/cmp-nvim-lsp', 'L3MON4D3/LuaSnip', 'saadparwaiz1/cmp_luasnip' },
   },
 
-  { -- Useful plugin to show you pending keybinds.
-    'folke/which-key.nvim',
-    config = function()
-      vim.o.timeout = true
-      vim.o.timeoutlen = 300
-      require('which-key').setup {}
-    end,
-  },
-
+  -- Useful plugin to show you pending keybinds.
+  { 'folke/which-key.nvim', opt = {} },
```

Comment:
Should be `opts`. Same for a bunch of others below :)

Reply:
good call haha